proposals/0038-16bit-isspecialfloat.md

```diff
@@ -108,6 +111,8 @@ not enabled.
 * How will the execution results be tested?
 There are existing SM6.2 HLK tests for 16 bit float 'isinf', 'isnan',
 and 'isfinite'. The plan is to copy these tests for the SM6.9 HLK.
+A new HLK test for 16 bit float 'isnormal' will need to be written and added
+to the SM6.9 HLK.
```

Comment:
The existing 32-bit `isnormal` test should also be modified to remove the IR modification and use the HLSL intrinsic instead.